ludwig/schema/combiners/comparator.py

```diff
@@ -7,15 +7,22 @@
 from ludwig.schema.metadata.combiner_metadata import COMBINER_METADATA
 
 
-@dataclass
+@dataclass(repr=False)
 class ComparatorCombinerConfig(BaseCombinerConfig):
     """Parameters for comparator combiner."""
 
     entity_1: List[str]
-    """TODO: Document parameters."""
+    """List of input features that compose the first entity to be compared."""
 
     entity_2: List[str]
-    """TODO: Document parameters."""
+    """List of input features that compose the second entity to be compared."""
+
+    type: str = schema_utils.StringOptions(
```

Comment:
ah yes, wanted this for a long time - nice!

Reply:
Thanks!